wagtail_localize/models.py

```diff
@@ -1511,7 +1511,7 @@ def get_field_path_from_stream_block(stream_value, path_components):
                         ]
 
                         if isinstance(block_def, blocks.StructBlock):
-                            return [block.block_type, path_components[1]]
+                            return [block.block_type] + path_components[1:]
```

Comment:
This change ensures we can process a chooser (e.g. image) in a StructBlock in a StructBlock in a StreamBlock in a StreamBlock


e.g.

```python
class ImageBlock(StructBlock):
    """
    Custom `StructBlock` for utilizing images with associated caption and
    attribution data
    """
    image = ImageChooserBlock(required=True)
    caption = CharBlock(required=False)


class Card(StructBlock):
    image_struct_block = ImageBlock()
    page = PageChooserBlock()
    text = CharBlock(required=False)


class PageInStreamBlock(StreamBlock):
    text = CharBlock(required=False)
    page_link = PageChooserBlock()
    card = Card()


class StoryBlock(StreamBlock)

    pages_in_streamblock = PageInStreamBlock()
    nested_stream = StreamBlock([
        ('pages_in_streamblock', PageInStreamBlock()),
    ])
```

Reply:
It looks like this would work for a structblock in a structblock because the field and content paths are the same for struct blocks.

If there's a streamblock within a structblock, would the content paths be converted to the correct field paths?

Reply:
Note: tidied up in 1e284bb (during our pair programming session)

also added a test